build.gradle

```diff
@@ -91,7 +101,13 @@ task createWindowsInstaller(type: Exec, dependsOn: 'createExe') {
     }
 }
 
-void buildEverything() {
+task buildEverything {
```

Comment:
Don't need to do this, you can just directly convert `buildEverything()` to a task

Reply:
OK, I fixed it